lib/uplink-gomobile/test-sim.sh

```diff
@@ -12,9 +12,14 @@ then
       echo "ANDROID NDK is not available" && exit 1
 fi
 
+if [[ -z "$STORJ_NETWORK_HOST4" ]] || [[ $STORJ_NETWORK_HOST4 == "127.0.0.1" ]]
```

Comment:
The code below defaults to `127.0.0.1` when it's not set, so it shouldn't need the check.

Reply:
The issue is that for this test `storj-sim` cannot use `127.0.0.1` because emulator will not connect. This is check for the case if someone will setup `STORJ_NETWORK_HOST4` externally to `127.0.0.1`.

Reply:
Let's start the emulator on a different loopback address, then there shouldn't be any conflicts.

Reply:
Yeah, I'm unable to find the right flags. :/

Reply:
My plan is to use `10.0.2.2` (emulator can access development machine with this address) and somehow set that as external address for `storj-sim` storagenodes.

Reply:
Maybe, if we add to `/etc/hosts`:
```
127.0.0.10   satellite.test
```

Then the android emulator should pick it up and allow connecting to `satellite.test` instead of the IP?

Reply:
and set storj-sim `host` to `satellite.test`?

Reply:
Hmm, the host should probably be `127.0.0.10`... not sure

Reply:
but storgenode address will be still `127.0.0.10:port` and this address satellite will pass to gomobile libuplink.

Reply:
you can modify storj-sim configs prior to running, or actually using `satellite.test` might actually work

Reply:
I'm just testing changing external address for storagnodes, later I will try next workarounds.

Reply:
hmm `contact.external-address` wont work because then the satellite will not see storage nodes, `/etc/hosts` also because its [not used by emulator](https://developer.android.com/studio/run/emulator-networking#dns)

Reply:
oh well then... let's hope we can move this to a completely separate node, so there's no concurrency issues

Reply:
I have one more idea, I will try to modify development machine `hosts` and emulator `hosts` to point the same address to different IPs.

Reply:
In theory, I added to my local `/etc/hosts` `127.0.0.1  storj-sim.test`, to emulator `/etc/hosts` `10.0.2.2 storj-sim.test` and set storage nodes external address to `storj-sim.test:port`. Mobile uplink still fails with `segment error: metainfo error: rpc error: code = Internal desc = not enough nodes: requested 10 found 0; {FreeBandwidth:17043968 FreeDisk:17043968 AuditCount:0 UptimeCount:0 ExcludedNodes:[] ExcludedIPs:[] MinimumVersion: OnlineWindow:4h0m0s DistinctIP:false}`

I'm not sure if my approach is wrong or I configure something incorrectly :/